.ci/bump-license-year-in-all-projects.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+set -e
+
+CURR_YEAR=$(date +"%Y")
+JAVA_FILE=$(find ./src/main -type f -name *.java -print -quit)
+echo "Current year will be taken from $JAVA_FILE"
+PREV_YEAR=$(grep "Copyright" $JAVA_FILE | cut -d " " -f 4 | cut -d '-' -f 2)
+echo "CURR_YEAR=$CURR_YEAR"
+echo "PREV_YEAR=$PREV_YEAR"
+
+./.ci/bump-license-year.sh $PREV_YEAR $CURR_YEAR .
+git add . && git commit -m "minor: bump year to $CURR_YEAR" && git push origin master
```

Comment:
Do you think we should have some pause and verify with a warning before doing the actual push? If diff is not a single line there is not way to stop the push.

Reply:
I do not think so if some thing will not work well it should be just fixed.
I used some validation when I was not sure in command, it was just sanity check.
It is not required if we start automate it.